Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -162,6 +162,16 @@
     <string name="osd_messages">Show On-Screen Display Messages</string>
     <string name="osd_messages_description">Display messages over the emulation screen area. These messages include memory card writes, video backend and CPU information, and JIT cache clearing.</string>
 
+    <!-- Path Settings -->
+    <string name="paths_submenu">Paths</string>
+    <string name="search_subfolders_ISO">Search Subfolders for Game Files</string>
```

Comment:
This string is not used.

Reply:
Removed. Search subfolder's setting seems to do nothing so that will be saved for another PR.